service/league_service/league_service.py

```diff
@@ -45,7 +45,7 @@ async def initialize(self) -> None:
 
         # Listen for league events
         await self._mq_service.listen(
-            config.EXCHANGE_NAME, config.LEAGUE_REQUEST_ROUTING_KEY, self.handle_message
+            config.EXCHANGE_NAME, config.LEAGUE_UPDATE_ROUTING_KEY, self.handle_message
```

Comment:
This is the routing key that's used to publish an event when the league service has updated someone's league score, whereas here we are listening for the trueskill rating updates. Given that the keys in the config hat unfortunate values I see how it is tempting to just change to this one, but then we would listen to updates on the same routing key as we would publish updates to which sounds like a very bad idea ;)

Right now there are three keys like this in the config:
```
LEAGUE_REQUEST_ROUTING_KEY: "success.gameResults.create"
LEAGUE_UPDATE_ROUTING_KEY:  "success.rating.update"
LEAGUE_UPDATE_FAIL_ROUTING_KEY:  "failure.rating.update"
```
I would suggest updating the key and value names to
```
TRUESKILL_RATING_UPDATE_ROUTING_KEY: "success.rating.update"
LEAGUE_RATING_UPDATE_ROUTING_KEY:  "success.leagueRating.update"
LEAGUE_RATING_UPDATE_FAIL_ROUTING_KEY:  "failure.leagueRating.update"
```
and use the first one here and the other two when the league service publishes updates.

Feel free to replace "rating" with "score" or similar if you prefer.